practice-two/src/css/main.css

```diff
@@ -34,8 +35,19 @@
   display: none;
 }
 
-@media screen and (min-width: 1024px) {
+.partner-section {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: 27px;
+  padding: 32px 0;
+}
+.partner-logo {
+  height: 20px;
+}
+@media screen and (min-width: 481px) {
   .container {
+    min-width: 896px;
```

Comment:
Quite confused, I will ping you to discuss this one

Reply:
i was fix it!!!